packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/onPremise/onPremisePcc/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dedicatedCloud_datacenter_remote_vra_network": "Réseau VRA Datacenter on premises"
```

Comment:
i don't know this product but does "on premises" makes sense for our french users?

Reply:
"On premises" might sounds weird, but it is the keyword used to indicate to customer that it is the datacenter of its own, not the one located in OVH datacenter. This term is also used in pictures and configuration path